arm9/src/unk_02006D98.c

```diff
@@ -0,0 +1,2855 @@
+#include "global.h"
+#include "unk_02006D98.h"
+#include "MI_memory.h"
+#include "NNS_g2d.h"
+#include "filesystem.h"
+#include "heap.h"
+#include "mod63_021DB450.h"
+#include "palette.h"
+#include "poketool/pokegra/otherpoke.naix"
+#include "registers.h"
+
+const u8 UNK_020ECCE0[] = {
```

Comment:
Try to interpret these as the struct to which they are typecast. iirc structs are aligned to short on arm946e-s when the largest member data type is short or byte aligned.

Reply:
I tried that before and it indeed fixes the padding but I couldn't get `UNK_020ECD4C` to be ordered properly. Just retried and now it works :man_shrugging:

Reply:
I've noticed adding functions to a file sometimes changes the order of the symbols so maybe I just needed to finish decompiling first

Reply:
Yeah, that's a consequence of compiler flag `-ipa file`.